docker-compose.yaml

```
@@ -0,0 +1,18 @@
+version: '3.7'
+
+services:
+  web:
+    restart: always
+    build:
+      dockerfile: Dockerfile
+      context: .
+    expose:
+      - "8011"
+    ports:
+      - 8011:8888
+    volumes:
+      - ./sammy/:/code/sammy
```

This may have been copy-pasta'ed from whatever example you used, but wouldn't work based on directories in this repo. Looking at the Dockerfile I"m imagining what you want is

```suggestion
    volumes:
      - .:/code/
```